scripts/globals/items/shadow_lord_shirt.lua

```diff
@@ -0,0 +1,17 @@
+-----------------------------------------
+-- ID: 26517
+-- Item: Shadow Lord Shirt
+-- Teleports user to Castle Zvahl Keep
+-----------------------------------------
+
+function onItemCheck(target)
+    return 0
+end
+
+function onItemUse(target)
+	-- Teleports user to Castle Zvahl Keep
+	-- May need a retail capture to verify
+	-- https://ffxiclopedia.fandom.com/wiki/Shadow_Lord_Shirt states teleports next to HP#1
+	-- so I copied the homepoint teleport position
```

Comment:
I think we should use the teleport effect and global instead like other items of this nature. Thats the only thing I see off in this PR :+1:

Reply:
Done - thanks!